Index: libs/tiff-4.0.2/tools/tiffgt.c
===================================================================
diff -u -N -ra2b5af5669e3dd8cb9add488cf4416027b5fcd10 -ra9017b4d4d227a1f3dabf3105757d7cff8c738d3
--- tiff-4.0.2/tools/tiffgt.c	(.../tiffgt.c)	(revision a2b5af5669e3dd8cb9add488cf4416027b5fcd10)
+++ tiff-4.0.2/tools/tiffgt.c	(.../tiffgt.c)	(revision a9017b4d4d227a1f3dabf3105757d7cff8c738d3)
@@ -31,11 +31,16 @@
 #include <string.h>
 #include <unistd.h>
 
-#if HAVE_APPLE_OPENGL_FRAMEWORK
+#if HAVE_OPENGL_GL_H
 # include <OpenGL/gl.h>
+#endif
+#if HAVE_GLUT_GLUT_H
 # include <GLUT/glut.h>
-#else
+#endif
+#if HAVE_GL_GL_H
 # include <GL/gl.h>
+#endif
+#if HAVE_GL_GLUT_H
 # include <GL/glut.h>
 #endif
 
